notification-manager/src/main/resources/application.yaml

```diff
@@ -1,3 +1,16 @@
 spring:
   application:
-    name: notification-manager
\ No newline at end of file
+    name: notification-manager
+  mail:
+    host: smtp.gmail.com
+    port: 587
+    username: sukharevroman96@gmail.com
+    properties:
+      mail:
+        debug: true
```

Comment:
What does debug do?

Reply:
information about connection to mail server, authentication to my email and status of message